### PR TITLE
Virtual geometry release notes for 0.14

### LINKED ISSUE #1288: Write release notes for PR #10164: Meshlet rendering (initial feature)
https://github.com/bevyengine/bevy/pull/10164 needs release notes for the upcoming Bevy release!

Please reply below if you'd like to write these notes. 
While the author(s) of the PR often have the context, knowledge and motivation to draft the release notes for their feature, anyone can contribute release notes!

------

Release notes should:

1. Clearly motivate the change.
2. Be written in a way that is understandable by the average Bevy user: some programming background and a general understanding of games.
3. Show off the coolest features of the PR. Screenshots are awesome, but elegant APIs are also welcome!
4. If this was a perf-centric PR, quantify the performance improvements. Graphs and statistics work well for this.

We can help you revise the release notes: a rough draft alone is incredibly useful :)
Your expertise is invaluable for contextualizing the changes; we'll work with you to bring the technical writing up to par.

To submit your release notes, modify `..\release-content\0.14\release-notes\10164_Meshlet_rendering_initial_feature.md` and submit a PR.
In that PR, please mention this issue with the `Fixes #ISSUE_NUMBER` keyphrase so it gets closed automatically.

### LINKED ISSUE #1303: Write release notes for PR #12755: Meshlet continuous LOD
https://github.com/bevyengine/bevy/pull/12755 needs release notes for the upcoming Bevy release!

Please reply below if you'd like to write these notes. 
While the author(s) of the PR often have the context, knowledge and motivation to draft the release notes for their feature, anyone can contribute release notes!

------

Release notes should:

1. Clearly motivate the change.
2. Be written in a way that is understandable by the average Bevy user: some programming background and a general understanding of games.
3. Show off the coolest features of the PR. Screenshots are awesome, but elegant APIs are also welcome!
4. If this was a perf-centric PR, quantify the performance improvements. Graphs and statistics work well for this.

We can help you revise the release notes: a rough draft alone is incredibly useful :)
Your expertise is invaluable for contextualizing the changes; we'll work with you to bring the technical writing up to par.

To submit your release notes, modify `..\release-content\0.14\release-notes\12755_Meshlet_continuous_LOD.md` and submit a PR.
In that PR, please mention this issue with the `Fixes #ISSUE_NUMBER` keyphrase so it gets closed automatically.

### PR DESCRIPTION
**Pull request by JMS55**
* Fixes #1288 
* Fixes #1303 
* You probably want to crop and shrink the screenshot. A video might also be cool to show the changing LOD levels as you move closer or further away. It's just the meshlet example, with a much tighter shadow cascade to get nicer looking shadows since it's a static screenshot.

---

* I should probably improve the plugin docs more in the future and copy past most of the "getting started guide" I wrote here into it
* I continue to wish I had a better name than `meshlets` for the feature 😓

### CODE REVIEW

File: release-content/0.14/release-notes/10164_Meshlet_rendering_initial_feature.md [hunk 1]
```diff
@@ -1 +1,66 @@
...
+
+This feature is still a WIP, and comes with several constraints, so be sure to read the docs and report any bugs you encounter. Look forward to more performance improvements (and associated breaking changes) in future releases.
+
+This feature does not use GPU "mesh shaders". Older GPUs are supported, but not recommended, and may become unsupported in a future release.
+
+In addition to the below user guide, checkout:
+* [The Bevy example for this feature](https://github.com/bevyengine/bevy/blob/release-0.14.0/examples/3d/meshlet.rs)
+* [The technical deep dive article by the main author of this feature](https://jms55.github.io/posts/2024-06-09-virtual-geometry-bevy-0-14)
+
+![Example scene using meshlets](meshlets.png)
```
Comment (killercup) [inline release-content/0.14/release-notes/10164_Meshlet_rendering_initial_feature.md:13]: Maybe add a sentence saying what is shown in this screenshot :)

**Review by killercup — APPROVED** [review]

File: release-content/0.14/release-notes/10164_Meshlet_rendering_initial_feature.md (hunk 1 above)
Comment (alice-i-cecile) [reply release-content/0.14/release-notes/10164_Meshlet_rendering_initial_feature.md:13]:
```suggestion
![Two rows of colorful bunnies (hailing from Stanford) are lined up. On the right,. normal high-polygon geometry is shown. On the left, their meshes have been broken up into patchwork regions, each with a different color.](meshlets.png)
```

**Review by alice-i-cecile — COMMENTED** [review]
Just a couple of suggestions: overall I really like the writing here. Friendly, clear and aimed at a good technical level.

**Comment by BD103** [comment]
Could you please delete [these lines](https://github.com/bevyengine/bevy-website/blob/dd974e0e11abd810853679c51fc75bb22713a25f/release-content/0.14/release-notes/_release-notes.toml#L201-L205C1) in `_release-notes.toml`? That should fix CI :)

File: release-content/0.14/release-notes/10164_Meshlet_rendering_initial_feature.md [hunk 2]
```diff
@@ -1 +1,66 @@
-TODO
+After several months of hard work, we're super excited to bring you an experimental release of a new virtual geometry feature!
+
+This new rendering feature works much like Unreal Engine 5's Nanite renderer. You can take a very high-poly mesh, preprocess it to generate a MeshletMesh during build time, and then at runtime render huge amounts of geometry - much more than Bevy's standard renderer can support. No explicit LODs are needed - it's all automatic, and near seamless.
+
+This feature is still a WIP, and comes with several constraints, so be sure to read the docs and report any bugs you encounter. Look forward to more performance improvements (and associated breaking changes) in future releases.
```
[the remark this answers is not in the document]
Comment (JMS55) [reply release-content/0.14/release-notes/10164_Meshlet_rendering_initial_feature.md:5]: I actually did mean constraints here. The constraints are unrelated to the fact that the feature is WIP.

**Pull request merged**